.bazelci/presubmit.yml

```diff
@@ -26,6 +26,7 @@ tasks:
       - "//..."
       # These tests are currently incompatible with RBE
       - "-//tests/integration/override_targets"
+      - "-//tests/com/jvm/external/maven:OutdatedTest"
```

Comment:
Excluding this test from RBE for now since it looks like the RBE build doesn't allow connections to outside maven servers.